### DISCUSSION TITLE
There is no message, but there is still one message notification in my inbox.

### DISCUSSION THREAD

**Discussion by deliangyang** [community/community#133475] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

How to fix it, please

![image](https://github.com/user-attachments/assets/ad6a6b3f-54f7-452f-b51d-996ba5ca0fc7)

**Comment by Sachinsingh2002** [comment]
hi @deliangyang github notifications can get stuck or not update correctly sometimes,

you can  simply refreshing the github page can clear up notification issues

or sometimes logging out and then logging back in can reset notification statuses

and yeah ensure that your notification settings are correct and that there’s no hidden notification.

**Comment by ebndev** [comment]
Hi @deliangyang, thanks for flagging this!

I am keeping a log of this issue as other users have mentioned this as well. The GitHub product team will evaluate the feedback but will not be able to respond to every submission. You can keep an eye on our [public roadmap](https://github.com/github/roadmap/projects/1) to see the kinds of work we are focusing on for the coming year. Smaller fixes won’t be listed, but are often included when work is happening adjacent to it.

In the meantime, please see [this previous discussion](https://github.com/orgs/community/discussions/123129#discussioncomment-9451090) for possible solutions.

Feel free to ask questions on the original threads if you run into an issue.

Thank you for contributing and continuing to support other users in the GitHub Community Discussions.
